exercises/matrices/matrix-1d-cpp/test/golden-model-py/simple_file.json

```diff
@@ -0,0 +1,5 @@
+{
+    "fruit": "Apple",
+    "size": "Large",
+    "color": "Red"
+}
```

Comment:
Why this file is needed? Is it just a sample along with the json header?

Reply:
It's not needed, used as code sample to load json files in a test program using jsoncpp lib